### PR TITLE
feat(RFC): 0003 code teams

### PR DESCRIPTION
**Pull request by jbolda**
I think plenty of this is open to discussion, but this should present a good base to open from.

### CODE REVIEW

**Comment by AmrBinBashir** [comment]
I really like this separation of concerns. Each team will have to focus on their part of the code only and they will make sure that their code is well-written and tested. This will help to better maintain the project as it grows.  ❤

**Comment by tensor-programming** [comment]
Looks good to me. Given how busy I've been and everyone else has been, we need new blood and some kind of on boarding incentive to keep development moving steadily.

File: texts/0003-code-teams.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+As decisions are made, the team should create tasks to take the decision to completion. The tasks should be of a size that is achievable in a PR. This will help scope work to something that anyone could reasonably hop into the code with minimal familiarity and help out.
+
+Secondary responsibilities would be review the PRs and enable the docs to make it easy for someone to come in and contribute to that portion on the code. ("Docs" doesn't necessarily need to be something on tauri.studio.)
+
+To address the inflow and outflow of members, we will list all team members and "open positions". Each team should have 3+ members, 3 of which being the primary members. The primary members should have the most expectation of responsibility. The primary members are not lifetime, static positions. There is an expectation that they change with some frequency and not be a "reward" to my earned.
```
Comment (nklayman) [inline texts/0003-code-teams.md:25]: "reward" to ~~my~~ **be** earned

File: texts/0003-code-teams.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Members may step back as an "alumnus" member. All roles operate in the open so this is doesn't reduce access in any manner. Rather the intent is to signal that the time is not available for them to give real attention to the project. A way that one could temporarily or permanently step back without any guilt.
```
Comment (nklayman) [inline texts/0003-code-teams.md:27]: the open so this ~~is~~ doesn't

Rather, the (add comma)

File: texts/0003-code-teams.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Tauri Bundler
+- Webview (interfacing with and work to make certain it has the APIs we need)
+
+### Engineering Impact
+#### Will this increase or decrease build times, developer workload etc.?
+More hands should help spread the load and improve the velocity of the docs and code.
+
+### Security Impact
+#### What changes will this bring, if any, to the security model of CI, developers and users?
+This should allow us to bring in more people to help out. More eyes covering every corner soups help, and relieving some of the managerial duties from core should allow more attention to security.
```
Comment (nklayman) [inline texts/0003-code-teams.md:46]: every corner ~~soups help~~ **will result in more bugs being caught**

**Review by nklayman — CHANGES_REQUESTED** [review]
Looks good, I think I found a few grammar mistakes though :)

**Comment by jbolda** [comment]
> Looks good, I think I found a few grammar mistakes though :)

Thanks, can you tell I typed this up on my phone? 😆

File: texts/0003-code-teams.md (hunk 1 above)
Comment (nothingismagick) [inline texts/0003-code-teams.md:25]:
```suggestion
To address the inflow and outflow of members, we will list all team members and "open positions". Each team should have 3+ members, 3 of which being the primary members. The primary members should have the most expectation of responsibility. The primary members are not lifetime, static positions. There is an expectation that they change with some frequency and not be a "reward" to be earned.
```

File: texts/0003-code-teams.md (hunk 2 above)
Comment (nothingismagick) [inline texts/0003-code-teams.md:27]:
```suggestion
Members may step back as an "alumnus" member. All roles operate in the open so this is doesn't reduce access in any manner. Rather, the intent is to signal that the time is not available for them to give real attention to the project. A way that one could temporarily or permanently step back without any guilt.
```

File: texts/0003-code-teams.md (hunk 3 above)
Comment (nothingismagick) [inline texts/0003-code-teams.md:46]:
```suggestion
This should allow us to bring in more people to help out. More eyes covering every corner will result in more bugs being caught, and relieving some of the managerial duties from core should allow more attention to security.
```

**Review by nklayman — APPROVED** [review]

**Pull request merged**